docs/EN_US/HPCCSpark/SparkHPCC.xml

```diff
@@ -482,7 +482,7 @@
             <listitem>
               <para>Typically when a job is run on a Spark cluster, it will
               take ownership of all worker nodes. In a shared cluster
-              environment this may not be ideal. Using the SPARK_MASTER_OPTS
+              environment this may not be ideal. Using the SPARK_MANAGER_OPTS
```

Comment:
@Michael-Gardner please confirm this variable name change is valid

Reply:
Revert back to SPARK_MASTER_OPTS